JASP-Engine/JASP/R/classicalmetaanalysis.R

```diff
@@ -519,46 +514,32 @@ ClassicalMetaAnalysis <- function(jaspResults, dataset = NULL, options, ...) {
 }
 
 .metaAnalysisRegTestFill <- function(jaspResults, dataset, options) {
-  # Compute/get model
   rma.fit <- .metaAnalysisComputeModel(jaspResults, dataset, options, ready = TRUE)
   egger   <- metafor::regtest(rma.fit)
-  coef    <- metafor::coef.summary.rma(summary(egger$fit))
-  #if(options$includeConstant)
-  #  jaspResults[["regTestTable"]]$addRows(list(name  = .unv(rownames(coef)[[1]]),
-  #                                             pval  = coef[1,4],
-  #                                             lower = coef[1,5],
-  #                                             upper = coef[1,6],
-  #                                             test  = coef[1,3]))
-  jaspResults[["regTestTable"]]$addRows(list(name  = .unv(rownames(coef)[[2]]),
-                                             pval  = coef[2,4],
-                                             lower = coef[2,5],
-                                             upper = coef[2,6],
-                                             test  = coef[2,3]))
+  jaspResults[["regTestTable"]]$setData(list(name = egger$predictor, test = egger$zval, pval = egger$pval))
```

Comment:
I take it you dropped the `lower` and `upper` because they weren't specified by `addColumnInfo()`?

Reply:
Yeah they weren't actually used